_pages/ato/types.md

```diff
@@ -3,15 +3,16 @@ title: Types of ATO
 navtitle: Types
 ---
 
-There are three tracks of documentation available:
+There are four tracks of authorization available:
 
+* [Pre-authorization for internal government use](#pre-authorization)
 * [90-day Lightweight Authority to Operate (LATO)](#day-authorization)
 * [1-year Lightweight Authority to Operate (LATO)](#year-authorization)
 * [Continuous Authority to Operate (cATO)](#continuous-authorization)
 
 ### Pre-authorization
 
-You may operate without authorization if all of the following conditions are met:
+You may operate without further authorization, based on our approved pre-existing security authorization, if all of the following conditions are met:
```

Comment:
This is implying that something can operate without ATO indefinitely without an ATO as long as it abides by the following conditions, but "pre-authorization" implies that it is only temporary. Maybe we need a different name for it?